udfs/remote_udfs/nlp/README.md

```diff
@@ -0,0 +1,119 @@
+# Calling Google's Natural Language API Sentiment Analysis Example 
+
+Google's Natural Language API Sentiment Analysis inspects the given text and identifies the prevailing emotional opinion within the text, especially to determine a writer's attitude as positive, negative, or neutral.  
+
+To run this example, you will need the following APIs enabled:
+* [BigQuery API](https://cloud.google.com/bigquery/docs)
+* [Cloud Functions API](https://cloud.google.com/functions/docs)
+* [BigQuery Connections API](https://cloud.google.com/bigquery/docs/working-with-connections#enable_the_connection_service)
+* [Natural Language API](https://cloud.google.com/natural-language/docs/setup#api)
+
+## Running the example
+
+### Setting the environment variables 
+
+Replace the various environment variables below with your desired values.
+```
+PROJECT=your_project_id
+LOCATION=your_bigquery_dataset_location
```

Comment:
We are using the same location for BigQuery and CF. I would suggest to specify the location here e.g. us-east1. Otherwise user may use US for bigquery location, but not supported by CF.

Reply:
I added a comment when setting the variables to use specific regions (not multi region) for the purpose of the demo.